script/flatpak

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo
```

Comment:
if we use user flatpak-builder , lets add the flag --user to all flatpak command ?
4. Now you are ready to build & install the flatpak. Run `flatpak-builder --user --install --force-clean build dev.zed.Zed.json` from the root directory.

Reply:
Will update once I'm done some other changes.